src/ocean/core/BitArray.d

```diff
@@ -189,13 +191,8 @@ struct BitArray
     }
     body
     {
-        size_t mDim=len/32;
-        ptr[0..mDim] = rhs.ptr[0..mDim];
-        int rest=cast(int)(len & cast(size_t)31u);
```

Comment:
Well because the documentation requires that the source array has *at least* the same number of bits, which contradicts this in contract which says the source array must have exact number of bits as the destination one.